### DISCUSSION TITLE
Issue Projects: Views: Member Should Be Able To Add View That Is Only Visible For Themselves (Personal View)

### DISCUSSION THREAD

**Discussion by BenjaminYde** [community/community#52624] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Currently when a view is added to a github project, all members see the views.
It would be very handy that members can add their own personal views to a project of an organization.
Only the member that added the personal view would be able see that view. (because is a personal view)
On creation of a view you should be able to choose the option (Shared View / Personal View)

![image](https://user-images.githubusercontent.com/44135938/231678646-a58671b9-8790-40a5-8d76-79b27befeca6.png)

**Note**: I am not talking about moving issues from repo A to repo B, I am talking about Github Issue Projects.

**Comment by opegamanalitarok** [comment]
Currently, GitHub does not offer a way to move multiple issues from one repository to another in one operation. However, there are a few workarounds you can use:

1. Use the GitHub API: You can use the GitHub API to automate the process of moving issues from one repository to another. You can write a script in your preferred programming language that uses the API to fetch all the issues in repository A and create the corresponding issues in repository B.

2. Use third-party tools: There are several third-party tools that can help you move issues between repositories. Some popular ones include ZenHub, GitKraken, and Octobox.

3. Manually move issues: Although it can be time-consuming, you can manually move each issue from repository A to repository B by copying the issue content and pasting it into a new issue in repository B. Make sure to include all relevant details, such as the issue title, description, and any associated labels or milestones.

Hope this helps! 🙌🙌

**Reply by BenjaminYde** [reply]
Thanks for the detailed reply, but I am not talking about github repos, but github projects.
I updated the description with extra info.
